modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/group_selector/page.jsp

```diff
@@ -55,25 +55,9 @@ Set<String> groupTypes = groupSelectorDisplayContext.getGroupTypes();
 			<liferay-ui:search-container-column-text
 				colspan="<%= 3 %>"
 			>
-				<liferay-frontend:horizontal-card
-					text="<%= curGroup.getDescriptiveName(locale) %>"
-					url="<%= groupSelectorDisplayContext.getViewGroupURL(curGroup) %>"
-				>
-					<liferay-frontend:horizontal-card-col>
-						<c:choose>
-							<c:when test="<%= Validator.isNotNull(curGroup.getLogoURL(themeDisplay, false)) %>">
-								<clay:sticker>
-									<img alt="" class="sticker-img" src="<%= curGroup.getLogoURL(themeDisplay, false) %>" />
-								</clay:sticker>
-							</c:when>
-							<c:otherwise>
-								<liferay-frontend:horizontal-card-icon
-									icon="<%= groupSelectorDisplayContext.getGroupItemSelectorIcon() %>"
-								/>
-							</c:otherwise>
-						</c:choose>
-					</liferay-frontend:horizontal-card-col>
-				</liferay-frontend:horizontal-card>
+				<clay:navigation-card
+					navigationCard="<%= new GroupNavigationCard(curGroup, groupSelectorDisplayContext, request) %>"
```

Comment:
Only a question

when we select the site on other places we are using `clay:navigation-card`


<img width="1658" alt="image" src="https://user-images.githubusercontent.com/47524751/184361895-13ba60e4-2d38-4219-8957-a90051d081b8.png">

The idea to change it to this card was studied or it's a nonsense?

Reply:
Hi @ealonso 
I do not know if you saw this question because I did not mention you :(